docs/reference/concepts/04-hooks.mdx

```diff
@@ -335,7 +335,7 @@ class MyHook(Hook):
     def error(self, context: HookContext, error, hints: dict):
         ...  # code to run if there's an error during before hooks or during flag evaluation
 
-    def finally_after(self, context: HookContext, hints: dict):
+    def finally_after(self, context: HookContext, details: FlagEvaluationDetails, hints: dict):
```

Comment:
if you want to make it even better, than you could also change the type hint for `hints`
```suggestion
    def finally_after(self, context: HookContext, details: FlagEvaluationDetails, hints: HookHints):
```
this can be applied to the other methods too

and `from openfeature.hook import Hook` needs to be changed to `from openfeature.hook import Hook, HookHints`

Reply:
ty!